backend/app/core/celery.py

```
@@ -38,155 +43,204 @@ class TaskConfig(BaseModel):
+            AttendanceRecords.attendance_id,
+            AttendanceRecords.date,
+            AttendanceRecords.clock_in_time,
+            AttendanceRecords.clock_out_time,
+            AttendanceRecords.status,
+            AttendanceRecords.total_hours,
+            AttendanceRecords.overtime_hours
+        ).where(
+            AttendanceRecords.user_id == user_id,
+            AttendanceRecords.clock_in_time >= start_date,
+            AttendanceRecords.clock_in_time <= end_date,
+            AttendanceRecords.is_active == True
```

Ruff (0.12.2)

Avoid equality comparisons to `True`; use `AttendanceRecords.is_active:` for truth checks

Replace with `AttendanceRecords.is_active`

(E712)